src/campaigns/campaigns.controller.ts

```diff
@@ -118,6 +118,8 @@ export class CampaignsController {
       })
     }
 
+    if (!campaign) throw new NotFoundException('Campaign not found')
```

Comment:
Pretty sure, if no campaign is found by `findFirstOrThrow` this if block won't ever run due to the exception already being thrown by Prisma? 🤔 

https://www.prisma.io/docs/orm/reference/prisma-client-reference#findfirstorthrow:~:text=However%2C%20if%20the%20query%20does%20not%20find%20a%20record%2C%20it%20throws%20a%20PrismaClientKnownRequestError.